src/api/app/policies/user_policy.rb

```diff
@@ -6,4 +6,8 @@ def update?
   def show?
     user.can_modify_user?(record)
   end
+
+  def rss_access?
```

Comment:
This should be removed as it's not needed.

Reply:
Thanks! Something went wrong when resolving conflicts in the rebase, because that was removed in the previous PR.
Anyway, I've reset the branch with master and cherry-picked the new commit. Everything should be ok now.